### ISSUE TITLE
Include tagging functionality under the hood

### ISSUE THREAD

**Issue by waldoj** [statedecoded/statedecoded#286]
The tagging functionality that's long been baked into Virginia Decoded should be included. That doesn't mean providing end-user functionality to add or remove tags, but it should provide functionality to import tags provided within the XML at the time of import, and make sure that internal API functionality is something that somebody could use to hack together a script to add their own tags.

**Comment by krues8dr** [comment]
@waldoj can you give me access to your Virginia Decoded fork so I can backport tags?  Or else move this to `Future` ?

**Comment by krues8dr** [comment]
I don't see tags in the XML so I don't know what's supposed to happen here.

**Comment by waldoj** [comment]
Yeah, that's not something that I dealt with when I moved to using the native XML importer (instead of the bespoke SGML importer for Virginia). SGML no longer even exists, so this is mooted.

What I _was_ doing was taking the tens of thousands of tags from _Richmond Sunlight,_ looking at the code sections modified by each tagged bill, and any tag used at least 5 times on bills that affect a given section was presumed to apply to that section, resulting a big CSV of tags and section identifiers. Then I sucked that in as a part of the import process, which was indexed in the search process, resulting in much better search results.

(Just writing this made realize it's absurd that this file isn't available for public access. I [just added that to Richmond Sunlight](https://www.richmondsunlight.com/downloads/section-tags.csv)).

To replicate this functionality, I'd need to modify the XML prior to importing it into The State Decoded, stepping through the CSV to do so. Sounds like a pain in the ass. Until somebody else wants this functionality, I'm pushing this off to Future.
